docs/framework/whats-new/obsolete-types.md

```diff
@@ -624,7 +624,6 @@ ms.author: "ronpet"
 
 <a name="VBCompat"></a>
 ### Assembly: Microsoft.VisualBasic.Compatibility.dll
-  For information about migrating from Visual Basic 6, see [Visual Basic 6.0 Resource Center](https://msdn.microsoft.com/library/windows/desktop/ms788229).
```

Comment:
I told you to get rid of VB6 links, but in this case here because it's about VB6 compat, I'd just update to previous-versions. Sorry!